MyTindy/mytindy-app
# Search

### :dart:  What is been accomplished in this PR:
- [x] Implement the search.
- [x] make the delete button of the card work
- [x] takes the data from the products file. 
#103 
----
### :compass:   How to test it out?
Steps to test my PR:
1. `npm i`
2. make sure to set up your `env` file to have these params
3. cd `.\client\`
4. `ionic serve`
